### ISSUE TITLE
javascripting v1.9.1 always reports null/blank for solution value

### ISSUE THREAD

**Issue by jbuberel** [sethvincent/javascripting#50]
Installed node and javascripting tutorial this mornig on Windows 8 machine:

```
$ node --version
v0.10.35
```

I then installed the current (v1.9.1) release of the javascripting tutorial from http://nodeschool.io/#workshopper-list using the command:

```
npm install -g javascripting
```

I created a file, introduction.js, with the contents:

```
console.log('hello');
```

I then ran the verify command, the the output was:

```
# O-oh, something isn't working.
# But don't panic!
-------------------
## Check your solution:
Solution
===================

Your Attempt
===================
hello

Difference
===================
hello
```

Checked in with the nodeschool discussion at https://gitter.im/nodeschool/discussions, and martin verified that this is a problem introduced in the v1.9.1 release. He had me revert to v1.9.0:

```
npm i javascripting@1.9.0 -g
```

And now the verification is working correctly:

```
$ javascripting verify introduction.js
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@     YOUR SOLUTION IS CORRECT!     @@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
-------------------
# YOU DID IT!
```

**Comment by SomeoneWeird** [comment]
Oops! That's no good, and probably my bad. Does this happen for all exercises?

**Comment by jbuberel** [comment]
It was happening for at least the first 4 exercises in the javascripting tutorial.

**Comment by jbuberel** [comment]
I haven't seen the problem in the rest of the javascripting module after downgrading

Nor have I seen it in the git-it, or learnyounode modules.

**Comment by SomeoneWeird** [comment]
@jbuberel this is fixed in the latest commit, thanks!

**Comment by sethvincent** [comment]
@jbuberal this fix is now published to npm at version 1.10.1. If you could reinstall and check it out that would be great.

**Comment by jbuberel** [comment]
Verfied in my local environment after upgrade to 1.10.1:

```
jason_000@velocity-micro ~/js
$ javascripting verify test.js
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@     YOUR SOLUTION IS CORRECT!     @@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
-------------------
# YOU DID IT!
Anything between the parentheses of console.log() are printed to the terminal.
So this:
console.log('hello');
prints hello to the terminal.
```
